svc/FetchService.js

```diff
@@ -25,6 +29,18 @@ import {stringify} from 'qs';
 export class FetchService {
 
     autoAbortControllers = {};
+    appHeaders = {};
+
+    /**
+     * Set App-wide headers for all subsequent requests. App-wide headers are not
+     * sent with core framework requests.
```

Comment:
I'm thinking that an app would almost always need to set these up right at the top of its `initAsync()` method to ensure they are used for all app fetches. If that sounds right, we might want to include that as a pointer here in the doc string.

We should ensure @lbwexler thinks this is a worthy addition to the service overall, and discuss if it needs to support a closure to be useful in practice across more scenarios (i.e. if you wanted to include a timestamp or token from some auth API that might expire or something else like that).

Reply:
well that closure bit is really interesting.  Seems like just the kind of thing that might come along.

Would a more general thing be a way to attach a closure to pre-process *all* requests?  FetchService.preprocessRequest(fn) and (potentially) FetchService.postProcessRequest?